pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 python = ">=3.8, <4.0"
 
 # Required
-boto3 = "^1.20.32"
+boto3 = "^1.26.96"
```

Comment:
boto3 and botocore are constrained to lambda compatible versions on purpose. See https://github.com/aws/aws-sdk-pandas/issues/2017